extensions/sql-migration/src/api/azure.ts

```diff
@@ -728,7 +731,7 @@ export async function validateIrDatabaseMigrationSettings(
 			dataSource: sourceServerName,
 			userName: migration._sqlServerUsername,
 			password: migration._sqlServerPassword,
-			// to-do: use correct value of encryptConnection and trustServerCertificate
+			encryptConnection: encryptConnection,
```

Comment:
for `validateIR` for MI/VM, verify that both `encryptConnection` and `trustServerCertificate` are explicitly passed

Reply:
This is for the source tho right? So we should honor what the user has specified?

Reply:
yes, on the source side in this example we do honor what the user passes in